mwui-stencil/src/components/mw-tabs/mw-tabs.tsx

```diff
@@ -17,6 +18,21 @@ export class MwTabs {
   @Prop({ mutable: true, reflect: true }) selected: number | null = null;
   @State() tabs = [];
 
+  /**
+   * Emits an event when tab is changed
+   */
+  @Event({
+    bubbles: true,
+    cancelable: false,
+    composed: false,
```

Comment:
please add a name prop as well - we notices when compositing multiple components that have e.g. an event called clickEmitter - there is issues with the @listen decorator
should be  { name: mwTabsClickEvent }

Reply:
I renamed the emitter variable now. Or did you mean to give the event an `eventName`?

Reply:
i meant the eventName - but tabsClickEmitter is a better name for the event anyways 
so i'd suggest to additionally add the eventName prop